src/main/kotlin/fi/metatavu/metaform/server/controllers/AdminThemeController.kt

```diff
@@ -0,0 +1,50 @@
+package fi.metatavu.metaform.server.controllers
+
+import fi.metatavu.metaform.server.persistence.dao.AdminThemeDAO
+import fi.metatavu.metaform.server.persistence.model.AdminTheme
+import java.util.UUID
+import javax.enterprise.context.ApplicationScoped
+import javax.inject.Inject
+
+/**
+ * Controller for admin themes
+ */
+@ApplicationScoped
+class AdminThemeController {
+    @Inject
+    lateinit var adminThemeDAO: AdminThemeDAO
+
+    /**
+     * Creates a new admin theme
+     *
+     * @param id the id of the theme
+     * @param data the data in the theme
+     * @param name the name of the theme
+     * @param slug the slug of the theme
+     * @param creatorId who created this theme
+     * @param lastModifierId who last modified this theme
+     *
+     * @return new [AdminTheme] object
+     */
+    fun create(
+        id: UUID, data: String, name: String, slug: String,
```

Comment:
I'd probably put these into a row, so the code is more consistent with the style of the rest of the program. It's a preference but will be easier to pick up due to consistency.